manageiq-operator/README.md

```diff
@@ -74,11 +131,44 @@ spec:
   applicationDomain: "miqproject.apps-crc.testing"
 ```
 
-## Creating an Operator Bundle
+## ManageIQ for OpenID-Connect Authentication Instance Example
 
-Create the bundle image and push to an image registry
+To run ManageIQ with OpenID-Connect Authentication, include these steps at **Step 4. Perform any optional custom configurations** from above.
+
+For this example we tested with Keycloak version 11.0
+
++ Step 4a. Create a secret containing the OpenID-Connect's `Client ID` and `Client Secret`
++ Step 4b. Modify the Custom Resource (CR) .yaml file to identify your OpenID-Connect provider URL and the secret just created.
+
+The details of these two steps are as follows:
 
+#### Step 4a. Create a secret containing the OpenID-Connect's `Client ID` and `Client Secret`
+
+You pick the name for `<name of your openshift client secret>`
+The values for `CLIENT_ID` and `CLIENT_SECRET` come from your Keycloak client definition.
+
+```bash
+$ oc create secret generic <name of your openshift client secret>  \
+  --from-literal=CLIENT_ID=<your Keycloak client ID>  \
+  --from-literal=CLIENT_SECRET=<your Keycloak client secret>
 ```
-$ operator-sdk bundle create docker.io/example/manageiq-bundle:0.0.1 --image-builder podman --directory deploy/olm-catalog/manageiq-operator/0.0.1/ --channels alpha --default-channel alpha
-$ podman push docker.io/example/manageiq-bundle:0.0.1
+
+#### Step 4b. Modify the Custom Resource (CR) .yaml file to identify your OpenID-Connect provider URL and the secret just created.
+
+Modify `deploy/crds/manageiq.org_v1alpha1_manageiq_cr.yaml` as follows:
+
+**Note:** The domain here will work for a Code Ready Containers cluster. Change it to one that will work for your environment.
+Additional parameters are available and documented in the Custom Resource Definition
+
+```yaml
+apiVersion: manageiq.org/v1alpha1
+kind: ManageIQ
+metadata:
+  name: miq
+spec:
+  applicationDomain: "miqproject.apps-crc.testing"
```

Comment:
The applicationDomain should probably be removed from here since it has its own section.

Reply:
I would rather leave it as a complete stand alone example of what the yaml could look like for an OIDC configuration.

Reply:
I'll remove the repeat of the note but would like this to stand as a complete example.

Reply:
Is `applicationDomain` required to make OIDC work?

Reply:
> Is `applicationDomain` required to make OIDC work?

It was in the example @abellotti gave me to use as a template for mine, so I assume it is.

Reply:
@abellotti Can you help clarify?

Reply:
Yes, applicationDomain is used for configuring OIDC and others like SAML.  The apache configuration for mod_auth_openidc needs parameters like the ServerName of the app as well as the redirect URI that instructs the IDP on where to redirect to in the application after a successful authentication.

Reply:
Okay, then lets keep it here.